src/udp_wrap.cc

```diff
@@ -219,6 +221,35 @@ void UDPWrap::Bind6(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void UDPWrap::BufferSize(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  UDPWrap* wrap;
+  ASSIGN_OR_RETURN_UNWRAP(&wrap,
+                          args.Holder(),
+                          args.GetReturnValue().Set(UV_EBADF));
+
+  CHECK(args[0]->IsUint32());
+  CHECK(args[1]->IsUint32());
+  int size = static_cast<int>(args[0].As<Uint32>()->Value());
+
+  int err;
+  if (args[1].As<Uint32>()->Value() == 0)
+    err = uv_recv_buffer_size(reinterpret_cast<uv_handle_t*>(&wrap->handle_),
+                              &size);
+  else
+    err = uv_send_buffer_size(reinterpret_cast<uv_handle_t*>(&wrap->handle_),
+                              &size);
```

Comment:
It’s not important but we prefer `{ … }` when the if/else bodies span more than one line.